bulk_site_creator/templates/bulk_site_creator/new_job.html

```diff
@@ -175,7 +176,30 @@ <h5 class="modal-title" id="staticBackdropLabel">Confirm Creation</h5>
                     selector: 'td:first-child'
                 },
                 order: [[ 1, 'asc' ]]
-            } );
+            });
+
+            // For updating modal body text.
+            $(".bsc-create-btn").on( "click", function(e) {
+                const totalTableRowItems = table.rows().count();
```

Comment:
You would see the updated values. Here's a good description I [found here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const):

"_The const declaration creates block-scoped constants, much like variables declared using the [let](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) keyword. The value of a constant can't be changed through reassignment (i.e. by using the [assignment operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment)), and it can't be redeclared (i.e. through a [variable declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#declarations)). However, if a constant is an [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object) or [array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array) its properties or items can be updated or removed._" - [mdn web docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const)

Since these variables are not being reassigned, I made them const 🙂